website/addons/wiki/static/ace-markdown-snippets.js

```diff
@@ -0,0 +1,54 @@
+ace.define("ace/snippets/markdown",["require","exports","module"], function(require, exports, module) {
+"use strict";
+
+exports.snippetText = "# Markdown\n\
```

Comment:
@icereval Are there any modifications to the original snippets file here? Should this go in the `vendor` directory?